scripts/web/blueprints/lecture.py

```
@@ -0,0 +1,584 @@
+    dest = DEFAULT_LECTURE_CHUNKS
+    dest.parent.mkdir(parents=True, exist_ok=True)
+    if not dest.exists():
+        dest.write_text("", encoding="utf-8")
+
+    existing_ids: set[str] = set()
+    for line in dest.read_text(encoding="utf-8").splitlines():
+        if line.strip():
+            try:
+                existing_ids.add(json.loads(line)["id"])
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)